__tests__/utils/install-module.spec.js

```diff
@@ -12,18 +12,44 @@
  * under the License.
  */
 
+const fs = require('node:fs/promises');
```

Comment:
should I change this so the Node.js 12.x tests can run?
```suggestion
const fs = require('fs/promises');
```
(here and other files, so can't apply this suggestion on its own)

Reply:
Yes lets leave this out for now. Dropping node 12 would be a breaking change

Reply:
changed in 03c3cfe